cmd/prometheus/boring.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build boring
```

Comment:
Are you sure this works without `GOEXPERIMENT=boringcrypto`? I don't think introducing our own build tag is necessary.

Ref: https://go.dev/src/crypto/tls/fipsonly/fipsonly.go

Reply:
IIUC - it's actually there to prevent the boilerplate alertmanager CI build stuff from failing, as that all uses a version of Go in the Github runners that doesn't have boringcrypto. So shielding those is the utility of this.

Reply:
The docs say it does, but I can just add it back to be explicit